packages/flutter/lib/src/cupertino/menu_anchor.dart

```diff
@@ -0,0 +1,3136 @@
+  ///
+  /// Use this for adjustments of the menu placement.
+  ///
+  /// Increasing [Offset.dy] values of [alignmentOffset] move the menu position
+  /// down.
+  ///
+  /// If the [alignment] is an [AlignmentDirectional] AND the text direction is
+  /// [TextDirection.rtl], a larger [Offset.dx] component of [alignmentOffset]
+  /// moves the menu position to the left. Otherwise, a larger [Offset.dx] moves
+  /// the menu position to the right.
+  ///
+  /// This value is ignored if the menu is opened with an explicit position.
+  final ui.Offset? alignmentOffset;
+
+  /// The point on the menu surface that attaches to the anchor.
+  final AlignmentGeometry? menuAlignment;
+
+  /// The constraints to apply to the menu scrollable.
+  final BoxConstraints? constraints;
+
+  /// Whether the menu's cross axis should be constrained by the overlay.
+  ///
+  /// If true, when the menu is wider than the overlay, the menu width will
+  /// shrink to fit the overlay bounds.
+  ///
+  /// If false, the menu will grow to fit the size of its contents. If the menu
+  /// is wider than the overlay, it will be clipped to the overlay's bounds.
+  ///
+  /// Defaults to false.
+  final bool constrainCrossAxis;
+
+  /// Whether or not a tap event that closes the menu will be permitted to
+  /// continue on to the gesture arena.
+  ///
+  /// If false, then tapping outside of a menu when the menu is open will both
+  /// close the menu, and allow the tap to participate in the gesture arena. If
+  /// true, then it will only close the menu, and the tap event will be
+  /// consumed.
+  ///
+  /// Defaults to false.
+  final bool consumeOutsideTaps;
+
+  /// Whether or not swiping is enabled on the menu.
+  ///
+  /// When swiping is enabled, a [MultiDragGestureRecognizer] is added around
+  /// the menu button and menu items. The [MultiDragGestureRecognizer] allows
+  /// for users to press, move, and activate adjacent menu items in a single
+  /// gesture. Swiping also scales the menu panel when users drag their
+  /// pointer away from the menu.
+  ///
+  /// Disabling swiping can be useful if the menu swipe effects interfere with
+  /// another swipe gesture, such as in the case of dragging a menu anchor
+  /// around the screen.
+  ///
+  /// Defaults to true.
+  final bool enableSwipe;
+
+  /// The duration after which a long-press on the anchor button will open the
+  /// menu.
+  ///
+  /// When a menu is opened via long-press, the menu can be swiped in the same
+  /// gesture to select and activate menu items.
+  ///
+  /// When the inner menu button is disabled, [longPressToOpenDuration] should
+  /// be set to [Duration.zero] to prevent the menu from opening on long-press.
+  ///
+  /// Defaults to [Duration.zero], which disables the behavior.
+  final Duration longPressToOpenDuration;
+
+  /// {@macro flutter.widgets.RawMenuAnchor.useRootOverlay}
+  final bool useRootOverlay;
+
+  /// The padding to subtract from the overlay when positioning the menu.
```

Comment:
Maybe this is clearer:
```suggestion
  /// Padding inside the overlay between its boundary and the menu content.
  ///
  /// Defaults to EdgeInsets.all(8).
```

IMO the current version explains the property from the perspective of "how the widget calculates position" instead of from the users' perspective of "how it affects my layout".